go.mod

```diff
@@ -1,29 +1,26 @@
 module github.com/docker/compose/v2
 
-go 1.16
+go 1.17
 
 require (
-	github.com/AlecAivazis/survey/v2 v2.2.3
-	github.com/buger/goterm v1.0.0
+	github.com/AlecAivazis/survey/v2 v2.3.2
+	github.com/buger/goterm v1.0.3
 	github.com/cnabio/cnab-to-oci v0.3.1-beta1
 	github.com/compose-spec/compose-go v1.0.4
-	github.com/containerd/console v1.0.2
+	github.com/containerd/console v1.0.3
```

Comment:
Looks like we're also updating various dependencies here; were these needed for Go 1.17? (otherwise it's good practice to do this separately).

Reply:
Not needed, thought I could do both at once with a nice comment on the PR, can separate if needed